llabake/Hello-Books
# chore(components): change components to JSX files

#### What does this PR do?

This PR changes components from js file to jsx files

#### Description of Task to be completed?

* change components from js file to jsx file

#### How should this be manually tested?

* clone the repo
* cd into the project directory
* run `npm install` to install all the dependency
* run `npm run start-dev` to start the back-end server
* run `npm run dev` to start the front-end server
* react components should have jsx file endings

#### What are the relevant pivotal tracker stories?

[finishes #158239827]

#### Any background context you want to add?

N/A
#### Screenshots

N/A
